dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops

```diff
@@ -239,3 +239,7 @@ scheme.incorrect.error.useraction=The scheme type must be http, https, ws, or ws
 upgradeHttpToWebSocket.failed=CWWKH0055E: The jakarta.websocket.server.ServerContainer#upgradeHttpToWebSocket call failed due to {0}. 
 upgradeHttpToWebSocket.failed.explanation=The server could not upgrade the HTTP or HTTPS request to the ws/wss protocol.
 upgradeHttpToWebSocket.failed.useraction=Review the error to identify why the request failed to upgrade.
+
+maxmessagesize.exceeded=CWWKH0056E: The maxMessageSize for method {0} exceeded Integer.MAX_VALUE for class {1}.
+maxmessagesize.exceeded.explanation=The OnMessage annotation parameter maxMessageSize can only be set to a maxmimum value of Integer.MAX_VALUE as per the WebSocket2.2 spec.
```

Comment:
Missing space before 2.2.   Looking at this again.... I think we should also spell out specification.